_i18n/es/resources/user-guides/remote_node_gui.md

```diff
@@ -1,12 +1,11 @@
 {% assign version = '1.1.0' | split: '.' %}
-{% include disclaimer.html translated="false" version=page.version %}
-## Finding a node
-First things first, you need to find a node to connect to! [moneroworld.com](https://moneroworld.com/#nodes) has some great resources for finding nodes. One of the easiest methods
-would be to use a node run by moneroworld, but they have a tool for finding random nodes too.
+{% include disclaimer.html translated="true" version=page.version %}
+## Encontrando un nodo
+Primero lo primero, necesitas encontrar un modo para conectarte. [moneroworld.com](https://moneroworld.com/#nodes) cuenta con buenos recursos para encontrar nodos. Uno de los métodos más fáciles
+sería correr un nodo por moneroworld, pero también tienen una herramienta para encontrar nodos aleatorios.
 
-## Connecting to the node from the GUI wallet
-After you enter your password for your wallet, you will see a pop up that will give you the option to "use custom settings". Click on it. You will then be
-sent to the "Settings" page in the GUI. At this point you should see two text boxes to the right of a label that says "Daemon address". In the first box (the one to the left) you need to enter the address of the node that you want to
-connect to. This address might look like `node.moneroworld.com` or it could look like any old ip address. The smaller box to the right is where you enter the node's port. The default port is `18081` but if you are using a random node the port that is used will vary. The port for node.moneroworld.com uses 18089.
-### Your screen should look a bit like this
+## Conectándose al nodo desde el monedero GUI
+Después de que ingreses la contraseña de tu monedero, verás un pop-up que te dará la opción de usar ajustes personalizados (use custom settings). Haz clic en él. Entonces serás enviado a la página de ajustes en la GUI. En este punto debes ser capaz de ver dos cajas de texto en la derecha de una etiqueta llamada "Daemon address". En la primera caja (a la izquierda) necesitarás ingresar la dirección del nodo al que te deseas conectar. Esta dirección se debe ver como `node.moneroworld.com` o puede verse como cualquier dirección ip vieja. La caja pequeña en la derecha es donde ingresas el puerto del nodo. El puerto por defecto es `18081` pero si estás usando un nodo aleatorio el puerto que es usado puede variar. El puerto para node.moneroworld.com es 18089.
```

Comment:
I like more _ajustes personalizados_, I didn't know you've already translated a phrase like that. It can also be something like _usar configuración personalizada_.

Reply:
I go with:

"Usar configuración personalizada", it suits better :stuck_out_tongue_winking_eye: